yarn.lock

```diff
@@ -11702,15 +11702,20 @@ node-abi@^2.21.0, node-abi@^2.7.0:
   dependencies:
     semver "^5.4.1"
 
+node-addon-api@*:
```

Comment:
need to find out of the resulting crw theia yarn tarball will include BOTH node-addon-api 4.3.0 and 3.2.1, and if that will break anything (keytar, lerna).

Reply:
the similar state in upstream
- node-addon-api@*: https://github.com/eclipse-theia/theia/blob/f81ef73a5379c938f57aad5ba0a109964169b4a4/yarn.lock#L7560
- node-addon-api@^3.0.0 https://github.com/eclipse-theia/theia/blob/f81ef73a5379c938f57aad5ba0a109964169b4a4/yarn.lock#L7565